mod/photos.php

```diff
@@ -914,7 +914,9 @@ function photos_content(App $a)
 			'$submit' => DI::l10n()->t('Submit'),
 		]);
 
-		$usage_message = '';
+		$maximagesize_bytes = DI::config()->get('system', 'maximagesize');
```

Comment:
Can you please factor in the PHP upload_limit value and take the smaller of the two?

Reply:
Thank you very much for your feedback. Instead, I already tried via ´DI::config().get_cfg_var('upload_max_filesize')´ but ended but only on a complete white blank page when loading the upload side. Thus, I thought accessing PHP Vars might just not support for security reasons. Sorry, I still at building my dev setup and currently not able to debug efficiently to analyze such issues. :-)

Reply:
Remove the `DI::config().` prefix as it is a native function.

Reply:
Thanks, I'll try. I addition, I will look up error handling in PHP when arithmetic might is performed on non number types, e.g. `DI::config()->get('system', 'maximagesize');` returns `null`.

Reply:
No, it isn’t useful to the end user, you can remove it.